engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-base-subscriptions.controller.js

```diff
@@ -32,7 +32,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostBaseSubscriptions
             'id':                       $scope.$stateParams.contentHostId,
             'organization_id':          CurrentOrganization,
             'sort_by':                  'name',
-            'sort_order':               'ASC'
+            'sort_order':               'ASC',
+            'match_system':             true
```

Comment:
whats an example of a sub that can't be matched?  one that has had its quantity exhausted ?